packages/express/src/schematics/ng-add/ng-add.ts

```diff
@@ -0,0 +1,35 @@
+import { Rule, chain } from '@angular-devkit/schematics';
+import {
+  addDepsToPackageJson,
+  updateJsonInTree
+} from '@nrwl/schematics/src/utils/ast-utils';
+import {
+  expressTypingsVersion,
+  expressVersion,
+  nxVersion
+} from '../../utils/versions';
+
+function addDependencies(): Rule {
+  return addDepsToPackageJson(
+    {
+      express: expressVersion
+    },
+    {
+      '@types/express': expressTypingsVersion,
+      '@nrwl/express': nxVersion
+    }
+  );
+}
+
+function moveDependency(): Rule {
+  return updateJsonInTree('package.json', json => {
+    json.dependencies = json.dependencies || {};
+
+    delete json.dependencies['@nrwl/express'];
+    return json;
+  });
+}
+
+export default function() {
+  return chain([addDependencies(), moveDependency()]);
```

Comment:
We should invoke `@nrwl/node:ng-add` here so that they have the builders available.